src/transformers/models/whisper/generation_whisper.py

```diff
@@ -511,6 +511,7 @@ def generate(
 
         # 3. Make sure generation config is correctly set
         # Make sure the generation config is correctly set depending on whether timestamps are to be returned or not
+        org_gen_config_output_attention = bool(generation_config.output_attentions)
```

Comment:
Let's rename this to `return_attention`. Also, I don't think you need a boolean here, right?

```suggestion
        return_attention = generation_config.output_attentions
```